assets/javascripts/kitten/components/form/form-row.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import classNames from 'classnames'
+
+export class FormRow extends React.Component {
+  render() {
+    return (
+      <div className={ classNames('k-FormRow', this.props.className) }>
```

Comment:
Est-ce qu'on passerait pas `{ ...this.props }` à cet élément ?

Par contre il faut le mettre avant `className` (pour pouvoir forcer notre `className`).
Et on peut retirer `{ this.props.children }</div>` du coup (car `children` se trouve dans les props aussi).